irohad/main/config/impl/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_library(irohad-flags
```

Comment:
Please, move all this logic to irohad/main/config/CMakeLists.txt, so that there are no ../flags.hpp